bedrock/firefox/templates/firefox/new.html

```diff
@@ -93,12 +93,15 @@
     <div class="row">
       <img src="{{ static('img/firefox/new/firefox-logo.png') }}" alt="Firefox" />
       <h1 class="large">
-      {% if l10n_has_tag('fx10_independent') %}
+      {% if LANG == 'en-US' or l10n_has_tag('spring_release') %}
```

Comment:
Cory mentioned in IRC that the headline should eventually be served to all locales. Do we need the `LANG == 'en-US'` check in this case?